doc-examples/src/main/scala/geotrellis/doc/examples/spark/VoxelKey.scala

```diff
@@ -66,6 +65,9 @@ object VoxelKey {
       (k, sk) => VoxelKey(sk.col, sk.row, k.z)
     )
   }
+
+  implicit val voxelKeyEncoder: Encoder[VoxelKey] = deriveEncoder[VoxelKey]
```

Comment:
Hm, would the annotation here more user friendly? (less lines ~ easier to understand)

Reply:
They didn't compile for some reason and this was the work-around. I didn't look into why they didn't because it seems like the annotation has been very fragile in other cases and using `deriveEncoder` is mostly the default in my mind.